daisy_workflows/build-publish/rhui/rhua_artifacts/rhua.patch

```diff
@@ -73,3 +73,16 @@ diff -Naur playbooks/rhua-provision.yml playbooks/rhua-provision.yml
  
      - name: Collect static browsable API and media files
        command: "{{ pulpcore_env }} pulpcore-manager collectstatic --noinput"
+@@ -402,6 +407,12 @@
+     - name: Set SELinux httpd_can_network_connect boolean and keep it persistent across reboots
+       command: /usr/sbin/setsebool -P httpd_can_network_connect on
+ 
++    - name: Allow drop-ins for NginX
++      lineinfile:
++        dest: "{{ template_prefix }}/nginx-pulp.conf"
++        insertafter: "http {"
++        line: "    include /etc/nginx/conf.d/*.conf;"
++
```

Comment:
i think we should change this to a *literal* patch to the file nginx-pulp, included in this file rhua.patch

this approach would add a step to the ansible workflow to patch its own template before use, which is much harder to reason about